packages/material-ui/src/MenuList/MenuList.js

```diff
@@ -6,66 +6,117 @@ import ReactDOM from 'react-dom';
 import warning from 'warning';
 import ownerDocument from '../utils/ownerDocument';
 import List from '../List';
+import getScrollbarSize from 'dom-helpers/util/scrollbarSize';
```

Comment:
```suggestion
import getScrollbarSize from '../utils/getScrollbarSize';
```

Reply:
@oliviertassinari I've been dealing with a flooded basement due to lots of rain on top of 2 feet of snow cover with the ground still frozen, so I've been a little distracted.